### ISSUE TITLE
add hostname to labels

### ISSUE THREAD

**Issue by 4bo** [stephenhillier/starlette_exporter#45]
```hostname``` is part of url, and an api may be configured for multiple domains.
so, adding it to labels of request count/time metrics could enrich it

**Comment by intelroman** [comment]
You can use on app.add_middleware(starlette_exporter.PrometheusMiddleware, app_name=os.environ['HOSTNAME']) 
in conjunction with env variables.
>>> import os
>>> print (os.environ['HOSTNAME'])
5516d03b1453

If Stephen think is good to have hostname I can take a look to add an optional filed with hostname from request header.

**Comment by intelroman** [comment]
> You can use on app.add_middleware(starlette_exporter.PrometheusMiddleware, app_name=os.environ['HOSTNAME']) in conjunction with env variables.
> 
> ```
> >>> import os
> >>> print (os.environ['HOSTNAME'])
> 5516d03b1453
> ```
> 
> If Stephen think is good to have hostname I can take a look to add an optional filed with hostname from request header.

Another easy way is to use. 
```
from fastapi import FastAPI
from starlette_exporter import PrometheusMiddleware, handle_metrics

app = FastAPI()
app.add_middleware(PrometheusMiddleware, app_name=Request.headers['host'])
app.add_route("/metrics", handle_metrics)
```

**Comment by stephenhillier** [comment]
Implemented in #52 and available in v0.14.0
